test/src/test/java/org/scenarioo/pizza/scenarioo/ScenariooBuildContext.java

```diff
@@ -0,0 +1,56 @@
+package org.scenarioo.pizza.scenarioo;
+
+import org.scenarioo.api.ScenarioDocuWriter;
+import org.scenarioo.model.docu.entities.Branch;
+import org.scenarioo.model.docu.entities.Build;
+
+import java.io.File;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import java.util.Date;
+
+
+public class ScenariooBuildContext {
+
+    private static final String BRANCH_NAME = "pizza-delivery-" + envOrDefault("BRANCH_NAME", "master");
+    private static final String BUILD_NAME = envOrDefault("BUILD_NUMBER", "undefined");
```

Comment:
I would like to see "build-" as prefix in front of the build name (just a number is a little bit strange maybe)

Reply:
OK